src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala

```diff
@@ -55,10 +55,8 @@ trait ClusterComponent extends LeoComponent {
       }
     }
 
-    // currently any string can be a GoogleProject
-    // but I'm planning to fix that soon
-    def getByName(project: GoogleProject, name: String): DBIO[Option[Cluster]] = {
-      clusterQuery.filter { _.googleProject === project }.filter { _.clusterName === name }.result flatMap { recs =>
+    def getByName(project: GoogleProject, name: ClusterName): DBIO[Option[Cluster]] = {
+      clusterQuery.filter { _.googleProject === project.value }.filter { _.clusterName === name.value }.result flatMap { recs =>
```

Comment:
The implicit conversions didn't work here.  I'm assuming this is because what's actually needed is a double implicit conversion, e.g. from `GoogleProject` -> `String` -> `Rep[String]`, and Scala doesn't do that.

So if we really wanted to make this prettier we'd need implicit conversions to `Rep[String]` too.